src/test/java/Homework19.java

```diff
@@ -0,0 +1,30 @@
+import org.junit.Assert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.annotations.Test;
+
+import java.util.Collection;
+
+public class Homework19 extends BaseTest {
+    @Test
+    public void deletePlaylist() throws InterruptedException {
+        String deletedPlaylistMsg = "Deleted playlist";
+        login("jose@test.com", "te$t$tudent");
+        openPlaylist();
```

Comment:
Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/227701460-0fa1bc55-5cda-4c5e-9765-2368168e9a64.png)